script/build.js

```diff
@@ -141,7 +142,7 @@ if (BUILD_OPTIONS.watch) {
 
 // TODO(awong): This URL needs to change based on target environment.
 smith.use(sitemap({
-  hostname: 'https://www.vets.gov',
+  hostname: hostnames[BUILD_OPTIONS.buildtype] || 'https://www.vets.gov',
```

Comment:
maybe this should just error out if the buildtype isn't present instead of failing silently

Reply:
I just realized that since `host` is an optional build argument, it might make sense to derive this property in `options.js` and use its default